### PR TITLE
fix(wallet)_: Error on sending ERC1155 collectible

### LINKED ISSUE #21183: "Objects are not valid as a React child" Error Displayed When Attempting to Send Multi-Collectible
### Steps:
1. Select a multi-collectible.
2. Go to the sending flow.
3. Proceed with the sending process.

### Actual result:
The error "Objects are not valid as a React child" is displayed  when attempting to send Multi-Collectible

https://github.com/user-attachments/assets/c1b5af1d-b255-4162-ad08-376f64713c53

### Expected result:
The user should be navigated to the multi-collectible sending screen without errors.

### Devices:
- Pixel 7a, Android 13
- iPhone 11 Pro Max, IOS 17

### Logs:
[logs.zip](https://github.com/user-attachments/files/16872873/logs.zip)

### ENV:
Nightly 6 Sep, 2024

### PR DESCRIPTION
**Pull request by smohamedjavid**
fixes #21183

### Summary

This PR:
- fixes crash while sending an ERC115 collectible
- fixes routes are fetched if we enter 0 in the input

### Platforms

- Android
- iOS

#### Prerequisites: A profile with tokens and ERC1155 collectibles

- Open Status
- Navigate to the wallet tab
- Try sending an ERC1155 collectible
- Verify no error is encountered
- Navigate to send token (ETH/ERC20) by picking any
- Enter 0 in the input field
- Verify the routes are not fetched

status: ready

### CODE REVIEW

**Review by ulisesmac — APPROVED** [review]
Thanks for the fix @smohamedjavid !

File: src/status_im/common/controlled_input/utils.cljs [hunk 1]
```diff
@@ -15,6 +15,10 @@
   (:value state))
 
 (defn numeric-value
+  [state]
+  (or (parse-double (input-value state)) 0))
+
+(defn value-bn
```
Comment (vkjr) [inline src/status_im/common/controlled_input/utils.cljs:21]: Just for the sake of similarity I'd name this as `bn-value`, wdyt? Or `value-numeric`/ `value-double` for the function above if we will use suffixes

**Review by vkjr — APPROVED** [review]

File: src/status_im/common/controlled_input/utils.cljs (hunk 1 above)
Comment (smohamedjavid) [reply src/status_im/common/controlled_input/utils.cljs:21]: Yeah, that's correct. I will clean this up 🙏

**Comment by VolodLytvynenko** [comment]
hi @smohamedjavid thank you for PR. No issues from my side. PR is ready to be merged

**Pull request merged**